src/controllers/dataset.ts

```diff
@@ -669,8 +669,24 @@ async function rebuildDatasetList(buildLogEntry: BuildLog, revisionList: Revisio
     error: string;
   }[] = [];
 
+  const buildScript = {
+    current_build: null as string | null,
```

Comment:
undefined should be ok here - `null as string` is a bit ming.

```suggestion
    current_build: undefined,
```